Foundation/JSONEncoder.swift

```diff
@@ -64,6 +64,9 @@ open class JSONEncoder {
 
     /// The strategy to use for encoding `Data` values.
     public enum DataEncodingStrategy {
+        /// Defer to `Data` for choosing an encoding.
+        case deferredToData
```

Comment:
Isn't adding the deferred case here both a change in ABI and also alphabetically unsorted?